src/nft/pages/collection/index.tsx

```diff
@@ -113,8 +114,8 @@ const Collection = () => {
           {contractAddress ? (
             <>
               {' '}
-              <Box width="full" height="276">
-                <Box width="full" height="276">
+              <Box width="full" height={`${COLLECTION_BANNER_HEIGHT}`}>
+                <Box width="full" height={`${COLLECTION_BANNER_HEIGHT}`}>
```

Comment:
I believe one of these divs can be removed

Reply:
u are right🙃 done